ch2-inventory/student-trungdc/02-nonFQDN.yml

```diff
@@ -0,0 +1,12 @@
+all:
+  children:
+    ungrouped:
```

Comment:
we don't need to EXPLICITLY have all/ungroup group defined. you can remove them to make your inventory shorter and more readable